ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh

```diff
@@ -153,16 +140,17 @@ ibm_cloud_cis_crn = "${IBMCLOUD_CIS_CRN}"
 ibm_cloud_tgw              = "${workspace_name}-tg"
 EOF
 
-  cp "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/var-mac-upi.tfvars "${SHARED_DIR}"/var-mac-upi.tfvars
-  cat "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/var-mac-upi.tfvars
+  cp "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/var-multi-arch-upi.tfvars "${SHARED_DIR}"/var-multi-arch-upi.tfvars
+  cat "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/var-multi-arch-upi.tfvars
 }
 
 function create_upi_powervs_cluster() {
   cd "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/ || true
   # Dev Note: https://github.com/ocp-power-automation/openshift-install-power/blob/devel/openshift-install-powervs#L767C1-L767C145
   # May trigger the redaction
   OUTPUT="yes"
-  ./openshift-install-powervs create -var-file var-mac-upi.tfvars -ignore-os-checks -verbose | sed '/.*client-certificate-data*/d; /.*token*/d; /.*client-key-data*/d; /- name: /d; /Login to the console with user/d' | \
+  /tmp/terraform apply -auto-approve -var-file var-multi-arch-upi.tfvars | \
```

Comment:
Failure from https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/56018/rehearse-56018-periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-powervs-ovn-multi-ppc64le/1829439528125861888/build-log.txt is at this line... 


Try
```suggestion
  ${IBMCLOUD_HOME_FOLDER}/terraform apply -auto-approve -var-file var-multi-arch-upi.tfvars | \
```